source/common/upstream/upstream_impl.cc

```diff
@@ -1051,6 +1052,9 @@ LegacyLbPolicyConfigHelper::getTypedLbConfigFromLegacyProto(
   return getTypedLbConfigFromLegacyProtoWithoutSubset(cluster, visitor);
 }
 
+using ProtocolOptionsHashMap =
+    absl::flat_hash_map<std::string, ProtocolOptionsConfigConstSharedPtr>;
```

Comment:
This alias could be moved to above and be used in the `parseExtensionProtocolOptions` method.